src/Umbraco.Cms.Integrations.SEO.GoogleSearchConsole.UrlInspectionTool/App_Plugins/UmbracoCms.Integrations/SEO/GoogleSearchConsole/URLInspectionTool/js/url-inspection-tool.controller.js

```diff
@@ -23,13 +24,17 @@
     });
 
     vm.onConnectClick = function () {
+        window.addEventListener("message", getAccessToken, false);
         vm.authorizationWindow = window.open(vm.oauthConfiguration.authorizationUrl,
             "GoogleSearchConsole_Authorize",
             "width=900,height=700,modal=yes,alwaysRaised=yes");
     }
 
     vm.onRevokeToken = function () {
         revokeToken();
+
+        vm.oauthSuccessEventCount = 0;
```

Comment:
I'm not quite following why this is an integer tracking a count, rather than a boolean.  We only seem to be checking for a value of `1`, so seems it could be `true|false`?

Reply:
Removed counter completely as it is not adding value due to the add/remove operations for event listener.